src/frontend/src/flows/verifiableCredentials/vcIssuer.ts

```diff
@@ -107,4 +109,51 @@ export class VcIssuer {
 
     return result.Ok;
   };
+
+  getDerivationOrigin = async ({
+    origin,
+  }: {
+    origin: string;
+  }): Promise<
+    | { kind: "origin"; origin: string }
+    | { kind: "use_default" }
+    | { kind: "error" }
+  > => {
+    const actor = await this.createActor();
+
+    const methodName = "derivation_origin" as const;
+
+    let result;
+    try {
+      result = await actor[methodName]({
```

Comment:
Why do you do it like this instead of `actor.derivatoin_origin`?

Reply:
mostly because this is brittle af because we have to check the error message; here at least if the method name ever changes then the error message check will get updated too